tutorials/hello/hello_client/test/unit/modules/hello/commands/create_hello_command.spec.ts

```diff
@@ -1,15 +1,48 @@
+import { testing, codec, cryptography, Transaction, chain, db, VerifyStatus } from 'lisk-sdk';
 import { CreateHelloCommand } from '../../../../../src/app/modules/hello/commands/create_hello_command';
+import { CreateHelloParams, createHelloSchema } from '../../../../../src/app/modules/hello/schema';
+import { ModuleConfig } from '../../../../../src/app/modules/hello/types';
+import { HelloModule } from '../../../../../src/app/modules/hello/module';
+import { CounterStore, counterKey } from '../../../../../src/app/modules/hello/stores/counter';
+import { MessageStore } from '../../../../../src/app/modules/hello/stores/message';
 
 describe('CreateHelloCommand', () => {
-  let command: CreateHelloCommand;
 
-	beforeEach(() => {
-		command = new CreateHelloCommand();
+	const getSampleTransaction = (params: Buffer) => ({
+		module: 'hello',
+		command: CreateHelloCommand.name,
+		senderPublicKey: Buffer.from("3bb9a44b71c83b95045486683fc198fe52dcf27b55291003590fcebff0a45d9a", 'hex'),
+		nonce: BigInt(0),
+		fee: BigInt(100000000),
+		params,
+		signatures: [cryptography.utils.getRandomBytes(64)],
+	});
+
+	let command: CreateHelloCommand;
+	let stateStore: any;
+	let counterStore: CounterStore;
+	let messageStore: MessageStore;
+
+	const config = {
+		"blacklist": [
+			"illegalWord1",
+			"badWord2",
+			"censoredWord3"
+		]
+	}
+
+	beforeEach(async () => {
+		const hello = new HelloModule();
+		command = new CreateHelloCommand(hello.stores, hello.events);
+		await command.init(config as ModuleConfig);
+		stateStore = new chain.StateStore(new db.InMemoryDatabase());
+		counterStore = hello.stores.get(CounterStore);
+		messageStore = hello.stores.get(MessageStore);
 	});
 
 	describe('constructor', () => {
 		it('should have valid name', () => {
-			expect(command.name).toEqual('createHello');
+			expect(command.name).toBe('createHello');
```

Comment:
toEqual should also work

Reply:
The vs code highlights this as an error when `toEqual` is used

Reply:
I would guess primitive types like string should use `toBe` instead, and use `toEqual` on comparing Objects and VS Code suggested to stick with this practice